packages/SwingSet/src/controller/controller.js

```diff
@@ -208,9 +208,8 @@ export async function makeSwingsetController(
     // Create an object with the property order that loadgen expects.
     const timedObj = { time: undefined, type: undefined, ...obj };
 
-    // Update the timedObj timestamp.
-    const timeMS = performance.timeOrigin + performance.now();
-    timedObj.time = timeMS / 1000;
+    // Update the timedObj timestamp, this uses unix gettimeofday()
+    timedObj.time = microtime.nowDouble();
```

Comment:
I would love if we could still keep a monotonic source in there. Is there any reason we couldn't include both `gettimeofday()` as `time` and `performance.now()` (without adding the origin, which becomes meaningless) as a second field?

```suggestion
    timedObj.time = microtime.nowDouble();
    timedObj.stamp = performance.now() / 1000;
```

That would require me to push an update to the loadgen, but that's pretty straightforward.